src/tests/bgfx.clj

```diff
@@ -311,7 +310,7 @@
                                     (-> (ToBytes)
                                         (GUI.HexViewer)))))
       (GUI.Window "Another window" :Pos (Float2 0.5 0.5) :Width 0.25 :Height 0.25
-                  :AllowMove true
+                  :Flags [GuiWindowFlags.NoResize, GuiWindowFlags.NoCollapse]
```

Comment:
We need better defaults

Reply:
That won't help here since you need to define one value anyway. Even if `NoResize`, `NoCollapse` and `NoMove` were defaults, you would still need to write that same line.